src/Bundle/Resources/config/services/metadata.xml

```diff
@@ -12,6 +12,10 @@
 -->
 
 <container xmlns="http://symfony.com/schema/dic/services" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+    <imports>
```

Comment:
Since `import` is introduced, all other "common" services from this file needs to be moved to sub-files as well.